components/menu/menu-item-styles.js

```diff
@@ -20,9 +23,9 @@ export const menuItemStyles = css`
 		color: var(--d2l-menu-foreground-color-hover);
 	}
 
-	/** separated from hover selectors because Safari <15.4 is having trouble parsing these */
-	:host(:focus-visible),
-	:host([first]:focus-visible) {
+	:host(.focus-visible),
```

Comment:
So could we not just remove menu's use of the `FocusVisiblePolyfill` mixin now and then also remove the `.focus-visible` class target?

Reply:
The `FocusVisiblePolyfill` isn't actually even used by menus, but I'd included the `.focus-visible` class for the case where focus is set programatically. But it turns out that somehow `:focus-visible` is working without that... I'm not entirely sure how. 🤯   It seems like it's safe to remove it.

I did a quick test that switches button to use `:focus-visible` for a dialog opener, and `:focus-visible` seems to work in that programatic case as well, at least in Chrome.

Reply:
Well, the visual-diffs at least benefit from the `.focus-visible` class. Maybe it is worth keeping.

Reply:
Hmm probably because they're calling `forceFocusVisible` which sets that class? I bet if they switched to just programatically call `.focus()` that could work.

Reply:
The tests are currently using standard `focus()`. I think they were new tests I'd added for my wcag 2.2 changes, and they effectively relied on `:focus`. I think there is some value in keeping them, but not sure we want to include `.focus-visible` to make them work.